src/devices/acurite.c

```diff
@@ -595,8 +597,10 @@ static int acurite_986_callback(bitbuffer_t *bitbuf) {
 	}
 
 	if ((status & 1) == 1) {
+            if (debug_output) {//All fprintf()'s must be under debug output guards.
 	    fprintf(stderr, "%s Acurite 986 sensor 0x%04x - %d%c: low battery, status %02x\n",
 		    time_str, sensor_id, sensor_num, sensor_type, status);
+            }
```

Comment:
Ugh.  No.   So you are going to make it so low battery messages only get seen when debug is enabled?

When and were do you feel exceptions should be reported?

Also I don't see any calls in this commit to change this to use the data printing stuff, so this is just making useful information disappear for no good reason.